test/unit/componentHandler.js

```diff
@@ -97,9 +97,9 @@ describe('componentHandler', function() {
 
   it('should upgrade a single component to an element by using its CSS classes', function() {
```

Comment:
Fixed test case 'should upgrade a single component to an element by using its CSS classes' to reveal a bug in the current upgradeElement caused by dataUpgraded not being correctly updated.